### PR TITLE
Window cleanup

### PR DESCRIPTION
**Pull request by emilwestergren**
Separated Window classes from OpenGL and could clean some code because of it.

### CODE REVIEW

File: ooc-x11.use [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Name: ooc UI
```
Comment (davidhesselbom) [inline ooc-x11.use]: You probably shouldn't use the same name as in `ooc-ui.use`...

File: ooc-x11.use [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+Description: User interface stuff
+SourcePath: source/ui/X11
```
Comment (davidhesselbom) [inline ooc-x11.use]: Shouldn't the folder be named with a lower-case `x`?

**Comment by emilwestergren** [comment]
Updated.

**Comment by emilwestergren** [comment]
Updated.

**Pull request merged**